locale/en/get-involved/index.md

```diff
@@ -16,6 +16,7 @@ The Node.js community is large, inclusive, and excited to enable as many users t
 - [Node.js Collection](https://medium.com/the-node-js-collection) is a collection of community-curated content on Medium.
 - [NodeUp](http://nodeup.com) is a podcast covering the latest Node news in the community.
 - The [Community Committee](https://github.com/nodejs/community-committee) is a top-level committee in the Node.js Foundation focused on community-facing efforts.
+- [The DEV Community Node.js tag](https://dev.to/t/node) is a place to share Node.js projects, articles and tutorials as well as start discussions and ask for feedback on Node.js-related topics. Developers of all skill-levels are welcome to take part.
```

Comment:
I agree with @fhemberger in adding `The DEV Community Node.js tag` to the learning section.

Reply:
@WaleedAshraf @fhemberger There's a medium collection which includes non official articles (many articles wasn't made by the official foundation). The DEV Community tag includes many discussions about Node too and it's a community so I think it would be probably best if it's under the community section. I hope you'll be convinced.

Reply:
@belhassen07 Yes, but the medium collection is maintained by members of the Node.js org.